MaterialLib/PorousMedium/UnsaturatedProperty/CapillaryPressure/VanGenuchtenCapillaryPressureSaturation.cpp

```diff
@@ -85,6 +89,41 @@ double VanGenuchtenCapillaryPressureSaturation::getdPcdS(
     const double val2 = std::pow(val1 - 1.0, -_m);
     return _pb * (_m - 1.0) * val1 * val2 / (_m * (S - _saturation_r));
 }
+
+double VanGenuchtenCapillaryPressureSaturation::getd2PcdS2(
+    const double saturation) const
+{
+    if (_has_regularized)
+    {
+        OGS_FATAL("Not implemented.");
+        /*
```

Comment:
What is the plan for the commented code? Is it a suggestion for the implementation?

Reply:
Fixing...